packaging/docker/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:slim
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN sed -i -e's/ main/ main contrib non-free/g' /etc/apt/sources.list.d/debian.sources \
+    && apt update \
+    && apt-get install -y netperf iperf3 iputils-ping wget \
```

Comment:
Flent doesn't support iperf3, so no reason to install that. Installing irtt would be a good idea, though... Why are you installing wget?

Reply:
Oh, I forgot to remove wget after experiments with just binary installation trying to reduce overall size.

Does flent support iperf2? In the documentation, it's mentioned that "some tests use iperf" without specifying what particular version.

Reply:
Yeah, there is some limited iperf2 support; I'm not against including that in the image, but it's kinda niche so not essential either...

Reply:
Thanks for the clarification, I'll remove it then.